src/psyclone/tests/psyad/main_test.py

```diff
@@ -319,6 +319,23 @@ def test_main_not_implemented_error(tmpdir, capsys):
     assert "This is not currently supported" in output
 
 
+def test_main_new_var(tmpdir):
+    '''Test that the main() function works when -a specifies a variable
+    name that does not exist in the original code but gets created as
+    part of the internal support for transformations.
+
+    '''
+    code = (
+        "program test\n"
+        "real :: a, b(10), c(10)\n"
+        "a = dot_product(b(:), c(:))\n"
+        "end program test\n")
+    filename = str(tmpdir.join("tl.f90"))
+    with open(filename, "w") as my_file:
```

Comment:
pylint complains about a missing `encoding='utf-8'` here.

Reply:
I changed all examples of this in the file